build/v1/types.go

```diff
@@ -946,6 +946,9 @@ type ImageChangeTrigger struct {
 	// will be used. Only one ImageChangeTrigger with an empty From reference is allowed in
 	// a build configuration.
 	From *corev1.ObjectReference `json:"from,omitempty" protobuf:"bytes,2,opt,name=from"`
+
+	// paused controls whether controller will trigger a build on an object change as well
```

Comment:
s/object/image/

Reply:
the doc for objectfieldtrigger.paused is clearer:
> // paused is true if this trigger is temporarily disabled. Optional.

would something like that be more accurate here?

Reply:
yeah i think that doc would be better.